### DISCUSSION TITLE
MeteoMedia [CA]

### DISCUSSION THREAD

**Discussion by Metroid2023** [iptv-org/community#928] [Q&A]
### Channel ID

MeteoMedia.ca

### Notes (optional)

Cable and Satellite feed.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
will be hard to have since its cable channel  .. only thing yet legal  is youtube channel vod

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19856
